labelme/app.py

```diff
@@ -41,7 +41,13 @@
 
 
 LABEL_COLORMAP = imgviz.label_colormap()
-
+gGroupIdColorMap = {0: (161,18,61), 1: (238,200,30), 2: (70,36,123), 3: (0,255,0),
+                          4: (119,103,158), 5: (238,255,0), 6: (113,126,17), 7:
+                            (0,255,238), 8: (30,0,255), 9: (20,0,220), 10: (0,191,255),
+                          11: (48,187,197), 12: (255, 0,162), 13: (211,146,208), 14:
+                            (128,0,0), 15: (160,115,137), 16: (71,144,154), 17: (128,105,181),
+                          18: (236,162,162), 19: (41,125,18), 20: (255,228,153), 21:
+                            (255,224,224), 22: (157,164,160), 23: (197,169,249)}
```

Comment:
@bayesian-mind we tried that with Victor and random colors sometimes were too similar :(. Therefore he fonds list of human "contrast" colors. Hope they are here.

Reply:
I think we can use random color generator as @bayesian-mind suggests in this particular module which will take out that chunk of code from there, and specify already applied gGroupIdColorMap in the ".labelmerc" file where all the user preferred configurations are stored! And in case user(annotator) won't have such configurations set up, random color generator will be used!